libmikmod/playercode/mplayer.c

```diff
@@ -2272,6 +2276,276 @@ static int DoOktArp(UWORD tick, UWORD flags, MP_CONTROL *a, MODULE *mod, SWORD c
 	return 0;
 }
 
+/*========== Farandole effects */
+
+static void DoFarTonePorta(MP_CONTROL* a)
```

Comment:
The `*` in the type name indicates an implied dereference of its associated identifier, i.e. `*a` is an `MP_CONTROL`, so the `*` should be attached to the identifier, not the type (`MP_CONTROL *a`). The rest of MikMod's code, including functions surrounding the ones you've added, seems to predominantly prefer this style.

Please review the code style of the surrounding code and update the new functions to match.

Reply:
Fixed the code styles etc. What about the global variables, should I move them into MP_CONTROL?